index.bs

```diff
@@ -304,22 +304,42 @@ it might still be possible; see [[#removing-features]].
 
 Do not assume that a change or removal is impossible without first checking.
 
-<h3 id=removing-features>Remove or change capabilities only once you understand existing usage</h3>
-
-Prioritize compatibility with existing content when removing or changing functionality.
-
-Once a significant amount of content has come to depend on a particular behavior,
-removing or changing that behavior is discouraged.
-Removing or changing features and capabilities is possible,
-but it first requires that the nature and scope of the impact on existing content
-is well understood.
-This might require research into how features are used by existing content.
-
-The obligation to understand existing usage also applies to any features that content relies upon.
+<h3 id=removing-features>Prioritize compatibility when changing or removing features</h3>
+
+Before changing how a feature behaves,
+fully understand how websites are currently using it.
+This might require research,
+for example by adding metrics to a popular user agent
+and/or by [searching the HTTP Archive](https://har.fyi/guides/getting-started/).
+Breaking content is harmful,
+and the benefit of a change has to
+significantly outweigh that harm to be worth doing.
+
+The obligation to understand existing usage
+also applies to non-standardized or unspecified features
+that content relies upon.
 This includes vendor-proprietary features and
-behavior that might be considered implementation bugs.
-Web features are not solely defined in specifications;
-they are also defined by how content uses those features.
+implementation bugs.
+
+Despite this, it is sometimes acceptable
+to break some existing content
+in order to help a large number of the web's users.
```

Comment:
Don't love the word "help"... wonder if there's a better word.

Reply:
```suggestion
in order to improve the experience for a large number of web users.
```